lib/config/types.ts

```diff
@@ -394,13 +394,7 @@ export interface RenovateOptionBase {
 
   name: string;
 
-  parent?:
-    | 'customDatasources'
-    | 'hostRules'
-    | 'packageRules'
-    | 'postUpgradeTasks'
-    | 'customManagers';
-
+  parents?: string[];
```

Comment:
should we create a more narrow type? 🤔

Reply:
I started with this but changed it after seeing the issue description. I think its better to use a narrower type, makes catching issues easier.
 
 ```ts
type AllowedParents = | 'customManagers' | 'customDatasources' | 'hostRules' | 'postUpgradeTasks' | 'packageRules';

parents?: AllowedParents[];
 ```

Reply:
Yes